inst/www/shared/shiny_scss/shiny.scss

```
@@ -92,6 +92,8 @@ pre.shiny-text-output {
   bottom: 0;
   left: 0;
   right: 0;
+  max-width: 100%;  /* Make sure width isn't restricted by styles on selectors like "body > div" */
+  max-height: 100%;
```

`max-*` won't work with something like `body > div { width: 100px; height: 100px }`

```suggestion
  width: 100%;  /* Make sure width isn't restricted by styles on selectors like "body > div" */
  height: 100%;
```